lib/auth/init.go

```diff
@@ -436,6 +444,25 @@ func isFirstStart(authServer *AuthServer, cfg InitConfig) (bool, error) {
 	return false, nil
 }
 
+// initClusterID initializes a unique id for this cluster. Cluster id is generated
+// and saved into the backend on the first start.
+func initClusterID(a *AuthServer) error {
+	data, err := a.bk.GetVal([]string{"cluster_id"}, "val")
```

Comment:
You can add cluster ID to `services.ClusterConfig`, then you can just access it via the `services.ClusterConfig` struct instead of directly.

Reply:
I agree with @russjones completely